### ISSUE TITLE
[Update suggestion] Spotify module for music player idea 

### ISSUE THREAD

**Issue by AzhamProdLive** [gh0stzk/dotfiles#123]
Hello there, it is I, with another idea for the spotify module for some polybars (Pamela being one of the examples)

After searching on internet and trying to see the best and worst possibilities, I passed by [this repo](https://github.com/PrayagS/polybar-spotify), which implement a Spotify module into polybars.

I am currently about to try it on my end, but I'm sure it would be a good addition/QoL update for some bars

**Comment by gh0stzk** [comment]
I'm going to check it

**Comment by gh0stzk** [comment]
Yep i will probably do it, with this repo https://github.com/mihirlad55/polybar-spotify-module or the one you mention. im still thinking. But that will need to wait, now im changing all eww config and making it works for all themes.

![Screenshot-22_09_2023-06-49-50](https://github.com/gh0stzk/dotfiles/assets/67278339/07879020-07e3-482b-85c9-75aa4dc5b79c)
![Screenshot-22_09_2023-06-50-06](https://github.com/gh0stzk/dotfiles/assets/67278339/5423eb49-743d-4ff1-8ac0-bb1f25b81a5a)

**Comment by AzhamProdLive** [comment]
Looks nice, and gotcha gotcha, if you need an external tester I'll be there

**Comment by gh0stzk** [comment]
Thank you i will let you know 😛
